### PR TITLE
[docs] CDCSDK: Documentation for CDC Consistent Snapshot streams

### PR DESCRIPTION
**Pull request by asrinivasanyb**
Documentation related to creation of Transactional CDC streams (consistent snapshot streams).

### CODE REVIEW

File: docs/content/preview/admin/yb-admin.md [hunk 1]
```diff
@@ -1682,7 +1682,32 @@ For example:
     create_change_data_stream ysql.yugabyte
 ```
 
+#### Creating a stream for Transactional CDC
 
+Creates a change data capture (CDC) DB stream for the specified namespace that can be used for Transactional CDC.
+This feature is {{<badge/tp>}}. The `yb_enable_cdc_consistent_snapshot_streams` flag can be used to enable the feature.
```
Comment (aishwarya24) [inline docs/content/preview/admin/yb-admin.md:1688]:
```suggestion
This feature is {{<badge/tp>}}. Use the [yb_enable_cdc_consistent_snapshot_streams](../../reference/configuration/yb-tserver/#yb-enable-cdc-consistent-snapshot-streams) flag to enable the feature.

```

**Review by aishwarya24 — COMMENTED** [review]
Is this even supported in stable (2.20) version? If yes, then we can back port it to the stable pages as well.

File: docs/content/preview/admin/yb-admin.md (hunk 1 above)
Comment (asrinivasanyb) [reply docs/content/preview/admin/yb-admin.md:1688]: Yes, this feature is in 2.20.2. Will backport these changes to stable pages as well

File: docs/content/preview/admin/yb-admin.md [hunk 2]
```diff
@@ -1682,7 +1682,32 @@ For example:
...
+```sh
+yb-admin \
+    -master_addresses <master-addresses> \
+    create_change_data_stream ysql.<namespace_name> EXPLICIT CHANGE USE_SNAPSHOT
+```
+
+* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *namespace_name*: The namespace on which the DB stream ID is to be created.
+* `EXPLICIT`: Checkpointing type on the server.
+* `CHANGE`: Record type indicating the server that the stream should send only the new values of the changed columns.
```
Comment (ddhodge) [inline docs/content/preview/admin/yb-admin.md:1701]:
```suggestion
* `CHANGE`: Record type indicating to the server that the stream should send only the new values of the changed columns.
```

File: docs/content/preview/admin/yb-admin.md [hunk 3]
```diff
@@ -1682,7 +1682,32 @@ For example:
     create_change_data_stream ysql.yugabyte
 ```
 
+#### Creating a stream for Transactional CDC
```
Comment (ddhodge) [inline docs/content/preview/admin/yb-admin.md:1685]:
```suggestion
##### Creating a stream for Transactional CDC
```

File: docs/content/preview/admin/yb-admin.md [hunk 4]
```diff
@@ -1682,7 +1682,32 @@ For example:
...
+yb-admin \
+    -master_addresses <master-addresses> \
+    create_change_data_stream ysql.<namespace_name> EXPLICIT CHANGE USE_SNAPSHOT
+```
+
+* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
+* *namespace_name*: The namespace on which the DB stream ID is to be created.
+* `EXPLICIT`: Checkpointing type on the server.
+* `CHANGE`: Record type indicating the server that the stream should send only the new values of the changed columns.
+* `USE_SNAPSHOT`: Snapshot option indicating intention of client to consume the snapshot. If the client is not interested in consuming the snapshot, the `NOEXPORT_SNAPSHOT` option is to be used.
```
Comment (ddhodge) [inline docs/content/preview/admin/yb-admin.md:1702]:
```suggestion
* `USE_SNAPSHOT`: Snapshot option indicating intention of client to consume the snapshot. If the client is not interested in consuming the snapshot, use the `NOEXPORT_SNAPSHOT` option.
```

File: docs/content/preview/admin/yb-admin.md [hunk 5]
```diff
@@ -1682,7 +1682,32 @@ For example:
     create_change_data_stream ysql.yugabyte
 ```
 
+#### Creating a stream for Transactional CDC
 
+Creates a change data capture (CDC) DB stream for the specified namespace that can be used for Transactional CDC.
```
Comment (ddhodge) [inline docs/content/preview/admin/yb-admin.md:1687]:
```suggestion
Create a change data capture (CDC) DB stream for the specified namespace that can be used for Transactional CDC using the following command.

```

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 6]
```diff
@@ -1070,6 +1070,14 @@ Default: `false`
 
 To learn about CDC, see [Change data capture (CDC)](../../../architecture/docdb-replication/change-data-capture/).
 
+##### --yb_enable_cdc_consistent_snapshot_streams
+
+Support for creating a stream for Transactional CDC is currently in [Tech Preview](/preview/releases/versioning/#feature-availability).
+
+Whether support for creation of streams for transactional CDC is enabled.
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:1077]:
```suggestion
Enable support for creating of streams for transactional CDC.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**